src/ethereum_test_fixtures/eof.py

```diff
@@ -53,3 +53,9 @@ class Fixture(BaseFixture):
     vectors: Mapping[Number, Vector]
 
     format: ClassVar[FixtureFormats] = FixtureFormats.EOF_TEST
+
+    def get_fork(self) -> str | None:
+        """
+        Returns the fork of the fixture as a string.
+        """
+        return None
```

Comment:
I'm wondering why this needs to return None? I think I am missing something

Reply:
The problem is that EOF container validation tests are actually fork-agnostic because they test the container of a certain version, which does not necessarily correlate to a fork, hence the `None`, but I agree this could be handled better.